testsuite/.rubocop_todo.yml

```diff
@@ -358,15 +360,11 @@ Style/DateTime:
     - 'features/step_definitions/datepicker_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
 
-# Offense count: 14
+# Offense count: 2
 # Cop supports --auto-correct.
 Style/DisableCopsWithinSourceCodeDirective:
   Exclude:
-    - 'features/step_definitions/command_steps.rb'
     - 'features/step_definitions/common_steps.rb'
-    - 'features/step_definitions/file_management_steps.rb'
-    - 'features/support/commonlib.rb'
-    - 'features/support/twopence_init.rb'
```

Comment:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/DisableCopsWithinSourceCodeDirective
I would maybe set it as `Enable: false`.

But it would be good to vote about it :)